pkg/backplaneapi/clientUtils.go

```diff
@@ -32,16 +32,19 @@ var (
 	DefaultClientUtils ClientUtils = &DefaultClientUtilsImpl{}
 )
 
-func (s *DefaultClientUtilsImpl) MakeRawBackplaneAPIClientWithAccessToken(base, accessToken string) (BackplaneApi.ClientInterface, error) {
-	co := func(client *BackplaneApi.Client) error {
+func makeClientOptions(accessToken string) BackplaneApi.ClientOption {
```

Comment:
Is there any reason to implement this function without a constructor pattern?
```
ex: 
func (s *DefaultClientUtilsImpl)makeClientOptions(accessToken string) ... {
 s.ClientOption = ....
}
```

Reply:
We talked and I cant do this as there is already another function in this file that also does not have a receiver which calls this function.